assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTile/index.js

```diff
@@ -248,16 +258,19 @@ export default function AudienceTile( {
 						topCities={ topCities }
 					/>
 
-					<AudienceTilePagesMetric
-						TileIcon={ AudienceMetricIconTopContent }
-						title={ __(
-							'Top content by pageviews',
-							'google-site-kit'
-						) }
-						topContentTitles={ topContentTitles }
-						topContent={ topContent }
-						isTopContentPartialData={ isTopContentPartialData }
-					/>
+					{ ( ( postTypeDimensionExists && isViewOnly ) ||
+						! isViewOnly ) && (
```

Comment:
Although this is as specced in the IB, I've realised the condition can be simplified. I'll apply the change myself to save the round trip.

```suggestion
					{ ( ! isViewOnly || postTypeDimensionExists ) && (
```